$MirOS: ports/games/frozen-bubble/patches/patch-frozen-bubble,v 1.1 2006/11/04 04:12:03 tg Exp $
--- frozen-bubble.orig	Thu Nov 23 21:45:23 2006
+++ frozen-bubble	Sat Dec 23 01:34:40 2006
@@ -48,7 +48,6 @@ use vars qw($TARGET_ANIM_SPEED $BUBBLE_S
 
 use Getopt::Long;
 use Data::Dumper;
-use Locale::gettext;
 use POSIX;
 use Math::Trig;
 use IO::File;
@@ -101,8 +100,6 @@ $KEYS->{misc}{$_} ||= $keys_orig->{misc}
 eval(cat_($hiscorefiles{levels} = "$ENV{HOME}/.fbhighscores"));
 eval(cat_($hiscorefiles{mptrain} = "$ENV{HOME}/.fbhighscores-mptrain"));
 
-textdomain("frozen-bubble");
-bind_textdomain_codeset("frozen-bubble", "UTF-8");  #- we're going to use SDL_Pango which uses UTF-8 input
 our $is_rtl = $ENV{LANGUAGE} =~ /^fa/;
 
 print "        [[ Frozen-Bubble-$version ]]\n\n";
